### ISSUE TITLE
Replies: Have a key for the reply text itself

### ISSUE THREAD

**Issue by turt2live** [matrix-org/matrix-doc#1541]
Currently people who wish to use replies need to parse either the fallback out of the reply or parse some HTML. In retrospect, the reply should probably have a key for the text that is being replied rather than forcing people to pick which key to throw parsing on.

For html-enabled clients this isn't really a problem, however for stuff like bridges the replies become hard to parse.

**Comment by Half-Shot** [comment]
Oh huh, I was about to look for this. Thank you!

**Comment by Half-Shot** [comment]
The other option here is to throw away the fallback and have non-compliant clients just show the reply text in the body, which would be space saving and avoid fallback screwups.
